sync/src/relayer/mod.rs

```diff
@@ -142,7 +142,25 @@ impl Relayer {
                 ban_time,
                 status
             );
+            metric!({
+                "topic": "error",
+                "tags": { "target": crate::LOG_TARGET_RELAY, "input": item_name, "status": format!("{:?}", status.code()) },
+                "fields": {},
```

Comment:
We can remove the entire `fields` key pair

Reply:
I think that keeping a consistent format is good for processing.
`"count"` can be omitted since different kinds of metrics could have their own fields;
But `"fields"` as a part of the metric format is better to show explicitly.

As for the log size, metrics logs are few. I don't think it brings the benefit that omits `{"fields": {}}`

Reply:
I guess we must have a program to convert the output and feed it to statistic DB? so the `fields: {}` actually not help for processing.

IMO it is not about the "consistent", normally we ignore the "target" key in logging macro, it is more likely for convenience and I believe nobody thinks its broken the consistency of the code. It is the same thing for the `metrics`, we just empty the field to a default value, we do not either break the `json` format or the scheme, just empty it. The `"fields": {}` in the source code and output log can be very confusing, most people will think some bug causes the output `{}`.

Reply:
`fields` is a part of the format.
Just like `cell_deps` is a part of a transaction format, we will keep `cell_deps: []` in the responded transaction in JSON even if it is empty.

Reply:
Update in d14635b4058b63b06a7d3dcaf2ad5488d33f8431:

* Fill `fields: {}` for metrics by default.